.gitignore

```diff
@@ -75,3 +75,4 @@ terraform/aks/vendor/
 Procfile.dev
 missing-location-graph-from-dttp.json
 .vscode/*
+Caddyfile
```

Comment:
I occasionally use this to fire up register with SSL but this can be ignored by the repo.